apps/greenhouse/apps/types/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@cloudoperators/juno-app-greenhouse-types",
```

Comment:
we should add at least private true so it does not get accidentally published

Reply:
👍